tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2Tests.cs

```diff
@@ -20,24 +20,24 @@
 
 namespace Datadog.Trace.ClrProfiler.IntegrationTests.CI
 {
-    public class MsTestV2Tests(ITestOutputHelper output) : MsTestV2TestsBase("MSTestTests", output, pre225TestCount: 19, post225TestCount: 22);
+    public class MsTestV2Tests(ITestOutputHelper output) : MsTestV2TestsBase("MSTestTests", output, pre224TestCount: 20, post224TestCount: 22);
 
-    public class MsTestV2Tests2(ITestOutputHelper output) : MsTestV2TestsBase("MSTestTests2", output, pre225TestCount: 19, post225TestCount: 21);
+    public class MsTestV2Tests2(ITestOutputHelper output) : MsTestV2TestsBase("MSTestTests2", output, pre224TestCount: 19, post224TestCount: 21);
 
     [Collection("MsTestV2Tests")]
     [UsesVerify]
     public abstract class MsTestV2TestsBase : TestingFrameworkTest
     {
         private readonly GacFixture _gacFixture;
 
-        public MsTestV2TestsBase(string sampleAppName, ITestOutputHelper output, int pre225TestCount, int post225TestCount)
+        public MsTestV2TestsBase(string sampleAppName, ITestOutputHelper output, int pre224TestCount, int post224TestCount)
             : base(sampleAppName, output)
         {
             TestBundleName = $"Samples.{sampleAppName}";
             TestSuiteName = "Samples.MSTestTests.TestSuite";
             ClassInitializationExceptionTestSuiteName = "Samples.MSTestTests.ClassInitializeExceptionTestSuite";
-            Pre225TestCount = pre225TestCount;
-            Post225TestCount = post225TestCount;
+            Pre224TestCount = pre224TestCount;
+            Post224TestCount = post224TestCount;
```

Comment:
The version did changed here because we all the integration changes I've added support of the missing spans to the 2.2.4 version of the nuget package. With these changes now the number of test spans we create in the integration matches the number of total tests from the console output.